packages/components/src/public/QueryModel/withQueryModels.tsx

```diff
@@ -434,7 +496,7 @@ export function withQueryModels<Props>(
 
             if (!isLoading) {
                 this.setState(
-                    produce<State>(draft => {
+                    produce<State>((draft: WritableDraft<State>) => {
```

Comment:
nit: Why is the explicit typing necessary here? Was there a TypeScript issue you were encountering that this rectified? This type is set correctly according to the generics as far as I can tell.

<img width="631" alt="image" src="https://github.com/user-attachments/assets/95b66e40-051d-4f9a-8f34-28b983016bba" />

Reply:
The entire file is full of red squigglies for me and Cory without these changes. Historically I've been able to clear my caches in Intellij and resolve the problem, but that is no longer working for me. If you know of a better long term solution I am all ears, but otherwise I think this is the best path forward.

Reply:
😢